spec/factories/factories.rb

```diff
@@ -0,0 +1,25 @@
+require './lib/pivotal_tracker'
```

Comment:
best practice is to have multiple .rb files in spec/factories directory.  So maybe `spec/factories/me.rb`?